src/stores/ScannerStore.js

```diff
@@ -286,6 +282,10 @@ export default class ScannerStore extends Container<ScannerState> {
 				multipartData: nextDataState
 			});
 		}
+
```

Comment:
could be moved together with line 281. :p

Reply:
that's in an else block, I want to update completedFramesCount every frame